src/index.html

```diff
@@ -1,13 +1,52 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="page">
+    <div class="page__card card" data-qa="card">
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <div class="card__subtitle">Product code: 195434</div>
+
+      <div class="card__reviews">
+        <div class="stars">
+          <div class="stars stars--4">
+            <div class="star_star"></div>
+            <div class="star_star"></div>
+            <div class="star_star"></div>
+            <div class="star_star"></div>
+            <div class="star_star"></div>
+          </div>
+        </div>
+
+        <div class="card__reviews-number">Reviews: 5</div>
+      </div>
+
+      <div class="card__price-area">
+        <div class="card__price-text">Price:</div>
+
+        <div class="card__price-number">$2,199</div>
+      </div>
+
+      <a
+        href="#"
+        class="card__button"
+        data-qa="hover"
+      >BUY
+      </a>
```

Comment:
follow code styling recommendations and write in lowercase (if you need your text to be uppercased - use styles)
```suggestion
      <a
        href="#"
        class="card__button"
        data-qa="hover"
      >
        buy
      </a>
```